packages/graphback-codegen-schema/tests/mock.graphql

```diff
@@ -38,3 +38,20 @@ type Query {
 type Mutation {
   likeNote(id: ID!): Note!
 }
+
+"""
+Model added to test schema generation using enum. 
+See issue https://github.com/aerogear/graphback/issues/1299
+
+@model
+"""
+type  Issue {
```

Comment:
I know this is not part of this but we might also add scalar here like: `rawContent`: JSON

Reply:
Ah sure. Can we create a separate issue for that `add more schema tests e.g scalar test, relationship tests`  this could be as simple as identify scalar types and annotations we have and use them here. Could be a good first issue too.